### PR TITLE
feat: add default value of projectRestrictedViewSize (DEV-2626)

### PR DESCRIPTION
**Pull request by mpro7**
### Task Description/Number

Issue Number: DEV-2626

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [ ] refactor: represents production code refactoring
- [x] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [ ] No

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]
LGTM

**Review by seakayone — CHANGES_REQUESTED** [review]

File: webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala [hunk 1]
```diff
@@ -100,6 +105,11 @@ final case class ProjectADMServiceLive(
       .map(_.map(_.ontologyMetadata.ontologyIri.toInternalIri))
       .map(_ :+ projectGraph)
   }
+
+  override def setProjectRestrictedViewSize(id: IriIdentifier, size: RestrictedViewSize): Task[Unit] = for {
+    project <- projectRepo.findById(id).someOrFail(NotFoundException(s"Project '${getId(id)}' not found."))
```
[the remark this answers is not in the document]
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala:110]: Apart from this it looks good to me.

[the remark this answers is not in the document]
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala:110]: In the responder you have already loaded the project, so there is also no need to query the database again here.

File: webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala [hunk 2]
```diff
@@ -100,6 +117,12 @@ final case class ProjectADMServiceLive(
       .map(_.map(_.ontologyMetadata.ontologyIri.toInternalIri))
       .map(_ :+ projectGraph)
   }
+
+  override def setProjectRestrictedViewSize(
+    id: IriIdentifier,
```
Comment (seakayone) [inline webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala:122]: This param is unused now, isn't it?

File: webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (mpro7) [reply webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala]: Thanks for your input. As we talked about in the responder only `ProjectADM` is exposed, thus it need an additional method that transform it into `KnoraProject`. Just for the recored :)

File: webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala (hunk 2 above)
Comment (mpro7) [reply webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/ProjectADMService.scala:122]: Oh my lord 🙃 It should be fine now.

**Pull request merged**